gulpfile.js

```diff
@@ -121,7 +122,7 @@ gulp.task('compile-tests', function () {
 
 gulp.task('default', function () {
     runSequence(
-        'watch',
-        $.shell.task('PATH=node_modules/.bin:$PATH electron .')
+        'clean',
+        ['typescript', 'sass', 'react']
```

Comment:
I understand but shell task don't added to gulp.watch so this may don't work on OS X